app/assets/stylesheets/common/base/compose.scss

```diff
@@ -390,47 +390,63 @@ html.composer-open {
 .autocomplete {
   z-index: z("composer", "dropdown") + 1;
   position: absolute;
-  width: 240px;
+  max-width: 300px;
+  width: 3000px;
```

Comment:
This looks odd to me. Isn't `3000px` a bit too much? :D

Reply:
Haha wow yes, that's supposed to be 300, good catch 😆